lib/interactive/control-router.js

```diff
@@ -187,4 +209,4 @@ ipcMain.on('manualButton', function(event, controlID) {
 });
 
 // Export Functions
-exports.router = controlRouter;
\ No newline at end of file
+exports.router = controlRouter;
```

Comment:
I'm not sure why random whitespace chacters are still getting stripped. I thought I turned that off in my IDE. Sorry about this!